docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -77,10 +100,11 @@ To see more details about each related process, select the process either in the
 [role="screenshot"]
 image::images/process-details.png[]
 
-See event details by selecting the **# events** URL at the top of the process details view or choosing one of the event pills in the graphical view.
+Loading process appear in a loading state when selecting that process for the first time. If loading data for a given process fails, simply click **Reload {process-name}** underneath the process to reload the data.
 
+See event details by selecting the **# events** URL at the top of the process details view or choosing one of the event pills in the graphical view.
 
-Events are categorized based on their `event.category`. 
+Events are categorized based on their `event.category`.
 
 [role="screenshot"]
 image::event-type.png[]
```

Comment:
Just a heads up these images might be broken. When I view this file directly in your branch they show up like this:

![image](https://user-images.githubusercontent.com/56361221/105748691-7c08d080-5f10-11eb-898e-492eae49590a.png)

Reply:
This should be good. These images appear in the preview link: https://security-docs_466.docs-preview.app.elstc.co/guide/en/security/master/visual-event-analyzer.html